### PR TITLE
Add python-lxml to docker images (and VMs)

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
For #25323 we need lxml as part of the docker images.

This relates to cmprescott/ansible-xml#52

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

test docker images

##### ANSIBLE VERSION

v2.4

### CODE REVIEW

**Comment by dagwieers** [comment]
And I guess we are going to need it on MacOSX and FreeBSD too.

**Review by gundalow — APPROVED** [review]

File: test/utils/docker/cloudstack-simulator/Dockerfile [hunk 1]
```diff
@@ -15,9 +15,10 @@ RUN apt-get -y update && apt-get install -y \
     netcat \
     openjdk-8-jdk \
     python-dev \
-    python-setuptools \
-    python-pip \
+    python-lxml \
```
Comment (mattclay) [inline test/utils/docker/cloudstack-simulator/Dockerfile:18]: This shouldn't be added. We don't run tests from this docker container.

Comment (dagwieers) [reply test/utils/docker/cloudstack-simulator/Dockerfile:18]: You are right, do I make a new PR, or will you revert ?

Comment (mattclay) [reply test/utils/docker/cloudstack-simulator/Dockerfile:18]: A new PR is fine.

Comment (dagwieers) [reply test/utils/docker/cloudstack-simulator/Dockerfile:18]: PR #25432 is ready, sorry for that.

**Pull request merged**